### PR TITLE
Add PHPCS

### PR DESCRIPTION
**Pull request by polybuildr**
Enabled some rules, they cause some failures. PR simply now. Once we want to seriously include this, I'll seriously look at the rules.

Edit: Fixed some issues and disabled some sniffs.

### CODE REVIEW

**Comment by polybuildr** [comment]
Also, /cc @hharchani @nisargjhaveri.

Should we show users with 0 score? What does everyone think? Pro: Everyone can see everyone from the start, so it's not like they're the only ones participating. Con: Lots of 0s. :P

File: src/app/models/sap_model.php [hunk 1]
```diff
@@ -7,7 +7,7 @@ public function __construct() {
 
     public function registerEntry($data) {
         // Optional form field
-        $organisedEvent = isset($data['organised-event']) ? $data['organised-event'] : NULL;
+        $organisedEvent = isset($data['organised-event']) ? $data['organised-event'] : null;
```
Comment (hharchani) [inline src/app/models/sap_model.php:10]: Yahan kyu hai ye?

File: src/app/controllers/sap.php [hunk 2]
```diff
@@ -53,7 +53,7 @@ public function login() {
             if (isset($_POST['csrf_token'])) {
                 $recvd_token = $_POST['csrf_token'];
             } else {
-                $recvd_token = NULL;
+                $recvd_token = null;
```
Comment (hharchani) [inline src/app/controllers/sap.php:56]: Why not `$recvd_token = isset($_POST['csrf_token'] ? $_POST['...'] : null;`

File: src/app/models/sap_model.php (hunk 1 above)
Comment (polybuildr) [inline src/app/models/sap_model.php:10]: Because lowercase `null`, `true`, `false` is a sniff.

File: src/app/controllers/sap.php (hunk 2 above)
Comment (polybuildr) [inline src/app/controllers/sap.php:56]: No reason. Not much for or against it I guess. And no point in this commit anyway. :P

**Pull request merged**